includes/WP_Admin.php

```diff
@@ -119,6 +120,11 @@ public static function initialize() {
 			ThemeInstallTaskManager::queue_initial_installs();
 		}
 
+		if(FlowService::initalize_flow_data())
+			FlowService::initalize_flow_data();
+		else
+			FlowService::get_default_flow_data();
+
```

Comment:
Not sure if this is gonna work cause the return value from `initalize_flow_data` is coming as `$updated_flow_data` which would always be there. To make sure this is saved in DB we probably need the return value from `self::update_wp_options_data_in_database($updated_flow_data);` in `initalize_flow_data`. Also not a good idea to call `initalize_flow_data` twice since these are costly operations.